generators/Exercises/Generators/PrimeFactors.cs

```diff
@@ -1,10 +1,18 @@
-﻿using Exercism.CSharp.Output;
+﻿using System.Linq;
+using Exercism.CSharp.Output;
 
 namespace Exercism.CSharp.Exercises.Generators
 {
     public class PrimeFactors : GeneratorExercise
     {
-        protected override void UpdateTestMethod(TestMethod testMethod) =>
+        protected override void UpdateTestMethod(TestMethod testMethod)
+        {
             testMethod.Input["value"] = (long)testMethod.Input["value"];
+
+            if (testMethod.Expected is int[])
```

Comment:
I'm on mobile, so that makes reviewing a bit harder. I think you can do "Expected is int[] expected", which allows you to remove the cast below.

Reply:
Nice I didn't know you could do that!